rpc-broker/src/json.c

```diff
@@ -0,0 +1,319 @@
+/*
+ Copyright (c) 2018 AIS, Inc.
+
+ This program is free software; you can redistribute it and/or modify
+ it under the terms of the GNU General Public License as published by
+ the Free Software Foundation; either version 2 of the License, or
+ (at your option) any later version.
+
+ This program is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU General Public License for more details.
+
+ You should have received a copy of the GNU General Public License
+ along with this program; if not, write to the Free Software
+ Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
+*/
+
+#include "rpc-broker.h"
+
+
+struct json_response *init_jrsp(void)
+{
+    struct json_response *jrsp = calloc(1, sizeof *jrsp);
+
+    jrsp->args = json_object_new_array();
+    // XXX rm and use dbus-message-get-serial
+    jrsp->id = rand() % 4096;
```

Comment:
what guarantees of uniqueness are required here and how confident this meets the requirement?